src/app/docs/key-info/key-info-docs.component.html

```diff
@@ -0,0 +1,164 @@
+<sky-docs-demo-page
+  moduleName="SkyKeyInfoModule"
+  moduleSourceCodePath="src/app/public/modules/key-info/"
+  pageTitle="Key Info"
+>
+  <sky-docs-demo-page-summary>
+    The key info component highlights important summary information.
+  </sky-docs-demo-page-summary>
+
+    <sky-docs-design-guidelines>
+
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p>
+          Use key info to highlight important summary information.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption=" Do use key info to highlight important summary information."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+      >
+        <p>
+          Use key info to show the total number of items in a list. For list totals, use the horizontal layout and place the key info at the top of the list.
+        </p>
+
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use key info to show the total number of items in a list."
+          captionType="success"
+          imageSource="~/assets/img/guidelines/key-info/key-info-usage-2.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use key info when a different data visualization is more appropriate for large amounts of data. For example, don't group multiple key info components to track how data changes over time. Use an alternative visualization such as a grid, bar chart, or line chart instead.
```

Comment:
I'd switch the first sentence to use plurals: `... when different data visualizations are more...`

I'd do the same in the third sentence: `Use alternatives such as grids, bar charts, or line charts instead.` And I'd make `grid` a link to the grid component docs.